src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -666,6 +666,23 @@ public virtual bool CanDetach()
 
         abstract public Task<List<ulong>> StartAddressesForLine(string file, uint line);
 
+        public virtual async Task<string[]> AutoComplete(string command, int threadId, uint frameLevel)
```

Comment:
This will need to be moved to `gdb.cs` and throw `throw new NotImplementedException();` here, `-complete` in lldb-mi responds with:
```
^error,msg="Driver. Received command '-complete b'. It was not handled. Command 'complete' not in Command Factory"
```